engines/csvlint/tests/test.csv

```diff
@@ -0,0 +1,4 @@
+"Foo","Bsr","Baz"
+
+"Biff","Baff","Boff"
+"Qux","Teaspoon","Doge"
```

Comment:
Missing new line on EOF.

Reply:
On purpose, it will emit one more error if added...

Reply:
Ah sorry, forgot this was a test case :P